api.go

```diff
@@ -32,9 +31,15 @@ const (
 	API_ERR_INTERNAL
 )
 
+type deploymentsResult struct {
+	deployments []DataDeployment
+	err         error
+}
+
 var (
-	deploymentsChanged = make(chan string)
-	addSubscriber      = make(chan chan string)
+	deploymentsChanged = make(chan interface{})
```

Comment:
Yes, I understand the semantics of the blocking behavior, I just don't feel it's an issue in practice. But it won't hurt anything either, so sure, I'll add a small buffer to them.